### PR TITLE
GitHub Actions Integration Tests

### PR DESCRIPTION
**Pull request by natenichols**
## High Level Overview of Change
Runs integration tests with github actions whenever a pull request is made against develop or master.

### Context of Change
This PR uses GitHub Actions to startup a rippled standalone node as a service. This node is configured to run an admin port on 0.0.0.0:6006, and is published to my personal docker hub. The integration tests then connect to this rippled standalone node to run integration tests against rippled. This PR does not change any of the tests, just how and when they are run. We should add more integration testing in the future.

### Type of Change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (non-breaking change that only restructures code)
- [x] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [ ] Documentation Updates
- [ ] Release

### CODE REVIEW

File: test/integration/integration-test.ts [hunk 1]
```diff
@@ -513,7 +505,7 @@ describe('integration tests', function () {
   it('generateWallet', function () {
     const newWallet = this.api.generateAddress()
     assert(newWallet && newWallet.address && newWallet.secret)
-    assert(isValidAddress(newWallet.address))
+    assert(isValidClassicAddress(newWallet.address))
```
Comment (mvadari) [inline test/integration/integration-test.ts:508]: Ah that's what the problem was - I was getting errors about this and wasn't sure why.

File: test/integration/integration-test.ts [hunk 2]
```diff
@@ -348,16 +350,6 @@ describe('integration tests', function () {
...
-  it('ticket', function () {
-    return this.api.getLedgerVersion().then((ledgerVersion) => {
-      return this.api
-        .prepareTicketCreate(address, 1, instructions)
-        .then((prepared) =>
-          testTransaction(this, 'ticket', ledgerVersion, prepared)
-        )
-    })
-  })
-
```
Comment (mvadari) [inline test/integration/integration-test.ts:360]: Why is this test removed?

**Review by mvadari — COMMENTED** [review]
Why are so many files deleted? 

Also, it looks like this PR should be merging into `github-actions` rather than `develop`

File: test/integration/integration-test.ts (hunk 2 above)
Comment (natenichols) [reply test/integration/integration-test.ts:360]: I can enable the amendment manually on my standalone node, and then this test will work.

**Comment by natenichols** [comment]
I'll rebase onto develop once the github-actions gets merged

**Comment by natenichols** [comment]
Those two tests are not running using mocha, I think that they're more scripts for manually testing. In the future, when we're shoring up integration tests we can write similar tests that use Mocha and can run against a standalone node.

**Comment by mvadari** [comment]
It'll automatically reset the base branch when github-actions is merged

**Review by intelliot — APPROVED** [review]
LGTM

**Review by mvadari — APPROVED** [review]

**Pull request merged**